int32 id

# Unknown
int64 unknown_num
geometry_msgs/Point unknown_center
geometry_msgs/Point[] unknown_voxels

# Free
int64 free_num
geometry_msgs/Point free_center
geometry_msgs/Point[] free_voxels

# Ground
geometry_msgs/Point[] ground_voxels

# Coverage Rate
float64 coverage_rate


